apps/examples/wc-vue3/vite.config.ts

```diff
@@ -3,5 +3,14 @@ import vue from '@vitejs/plugin-vue';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-    plugins: [vue()],
+    plugins: [
+        vue({
+            template: {
+                compilerOptions: {
+                    // Treat all tags containing dashes as custom elements
+                    isCustomElement: (tag) => tag.includes('-'),
```

Comment:
Is this approach recommended in any docs? If so, could you link them in a comment?